jekyll/asset/script/main.js

```diff
@@ -419,22 +444,40 @@
       postingData.requiredSkills = requiredSkills;
       postingData.preferredSkills = preferredSkills;
 
-      // Also include Certs
+      var certificationsNeeded = [];
+      $('input[name=certNeeded]').each(function() {
+        if(this.value) {
+          certificationsNeeded.push({name: this.value});
+        }
+      });
+      postingData.certificationsNeeded = certificationsNeeded;
 
       var trainingOfferedEl = $("input:radio[name=training]:checked");
       if(trainingOfferedEl) postingData.trainingOffered = trainingOfferedEl.val();
 
       return templates.fullJobPosting.render(postingData, templates);
     }
 
+    function collectAddedSkills() {
+      $('form + input').each(function(){
```

Comment:
This needs a more specific selector. We could end up adding a bunch of forms with inputs and it would look through them all for skills.